apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,240 @@
+                            <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M64 0C28.7 0 0 28.7 0 64V448c0 35.3 28.7 64 64 64H320c35.3 0 64-28.7 64-64V160H256c-17.7 0-32-14.3-32-32V0H64zM256 0V128H384L256 0zM216 408c0 13.3-10.7 24-24 24s-24-10.7-24-24V305.9l-31 31c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l72-72c9.4-9.4 24.6-9.4 33.9 0l72 72c9.4 9.4 9.4 24.6 0 33.9s-24.6 9.4-33.9 0l-31-31V408z"/>
+                        </svg>
+                         Folder Upload
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+        <!-- Dropdown buttons for User -->
+        <div>
+
+            <!-- Type Dropdown -->
+            <div class="dropdown dropdown-bottom">
+                <div tabindex="0" role="button" class="btn m-1 font-semibold normal-case px-6 border-white hover:border-white">Classification
+                    <span>
+                        <svg xmlns="http://www.w3.org/2000/svg" height="20" width="14" viewBox="0 0 320 512" fill="#fff">
+                            <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z"/>
+                        </svg>
+                    </span>
+                </div>
+                <ul class="dropdown-content z-[1] menu p-2 shadow-2xl bg-base-100 rounded-box w-52 border">
+                    <li>
+                        <a href="/">
+                            Open
+                        </a>
+                    </li>
+                    <hr>
+                    <li>
+                        <a href="/">
+                            Confidential
+                        </a>
+                    </li>
+                    <hr>
+                    <li>
+                        <a href="/">
+                            Secret
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <!-- Type Dropdown -->
+            <div class="dropdown dropdown-bottom">
+                <div tabindex="0" role="button" class="btn m-1 font-semibold normal-case px-6 border-white hover:border-white">Modified
+                    <span>
+                        <svg xmlns="http://www.w3.org/2000/svg" height="20" width="14" viewBox="0 0 320 512" fill="#fff">
+                            <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z"/>
+                        </svg>
```

Comment:
Save this svg into a file of it's own and use it dynamically via imports.

Reply:
Did the same for all other svg fixes